pkg/crc/machine/start.go

```diff
@@ -280,9 +281,6 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	if err := updateSSHKeyPair(sshRunner); err != nil {
 		return nil, errors.Wrap(err, "Error updating public key")
 	}
-	if err := copyKubeconfig(client.name, crcBundleMetadata); err != nil {
-		return nil, errors.Wrap(err, "Error copying kubeconfig file")
-	}
 
```

Comment:
We can do that, but why? Is this needed by one of the other commits? Is this fixing a bug? Is this just code movement? Would be easier to review if this was explicit :)

Reply:
I started with the kubeadmin-password but for completeness, I also did it for the ssh key and kubeconfig files.
The commit log should be better now.